src/main/java/no/nav/pto/veilarbportefolje/config/FeatureToggle.java

```diff
@@ -17,6 +17,8 @@ private FeatureToggle() {
 
     public static final String BRUK_FILTER_FOR_BRUKERINNSYN_TILGANGER = "veilarbportefolje.bruk_filter_for_brukerinnsyn_tilganger";
 
+    public static final String STOPP_INDEKSERING_ETTER_PDL_ENDRING = "veilarbportefolje.stopp_indeksering_etter_pdl_endring";
```

Comment:
Maybe just call this something more generic like `STOPP_INDEKSERING`. I think that communicates a bit better what it does - no need to mix in the fact that we are doing it this time because of PDL reload.

Reply:
```suggestion
    public static final String STOPP_INDEKSERING = "veilarbportefolje.stopp_indeksering";
```